src/p2p/net_node.inl

```diff
@@ -723,6 +723,11 @@ namespace nodetool
     {
       return get_ip_seed_nodes();
     }
+    if (not m_enable_dns_seed_nodes)
+    {
+      m_fallback_seed_nodes_added.test_and_set();
+      return get_ip_seed_nodes();
```

Comment:
You can set a domain through socks, so that the remote side does the lookup for the DNS seed nodes. This would require some more refactoring of the code though.

Could you add a comment about this here?